test/JDBC/upgrade/14_6/schedule

```diff
@@ -259,7 +259,6 @@ format
 msdb-dbo-syspolicy_system_health_state
 dateadd_internal_df
 sys-all_columns
-sys-all_sql_modules
```

Comment:
Please do not remove existing test files. If some modifications are needed due to this PR, then let's include that as well.

Reply:
Agreed